apps/public-reference/pages/applications/review/demographics.tsx

```diff
@@ -49,7 +49,13 @@ const Demographics = () => {
       defaultChecked: item.checked,
       register,
     }))
-  }, [])
+  }, [register])
+
+  const backPath = useMemo(() => {
+    return application.preferences.none
+      ? "/applications/preferences/general"
+      : "/applications/preferences/select"
+  }, [application.preferences.none])
```

Comment:
Why are you using `useMemo` here? Since the result is a string, it shouldn't matter for purposes of a dependency list in `useEffect` (string literal equality in JS is fixed if the string value is fixed), and this isn't an expensive operation - I'd just do this for simplicity:

```
const backPath = `/applications/preferences/${application.preferences.none ? "general" : "select"}`
```

Reply:
Agree, nice catch! 🥇